meerkat_abacus/data_management.py

```diff
@@ -347,9 +350,28 @@ def should_row_be_added(row, form_name, deviceids, start_dates, allow_enketo=Fal
             ret = False
         elif parse(row["SubmissionDate"]) < start_dates[row["deviceid"]]:
             ret = False
+    if ret:
+        ret = _validate_date_to_epi_week_convertion(form_name, row)
     return ret
 
 
+def _validate_date_to_epi_week_convertion(form_name, row):
+    form_data_types = data_types.data_types_for_form_name(form_name)
+    if form_data_types:
```

Comment:
One form can be involved in multiple data types and we need to figure out how to deal with this. Do we require all dates to be epi-weekable? To make things worse there are can also be conditions for if a row is actually a data type, we might have to check that too?

Reply:
I could find a better way to get a "epi_weekable" date for a form. I try to get data_types for the form and then test only vs 'date' field in it. If it's not present it's bypassed.
We could discuss a better approach.